qiskit/primitives/base/base_result.py

```diff
@@ -42,6 +43,13 @@ def __post_init__(self) -> None:
             TypeError: If one of the data fields is not a Sequence or ``numpy.ndarray``.
             ValueError: Inconsistent number of experiments across data fields.
         """
+        warn(
+            "The class ``BasePrimitiveResult`` is deprecated as of qiskit 1.0. "
```

Comment:
Could you update warning messages as well?
```suggestion
            "The class ``BasePrimitiveResult`` is deprecated as of qiskit 1.2. "
```